library/cwm/src/modules/CWM.rb

```diff
@@ -1026,6 +1060,30 @@ def validate_current_widgets(event)
     publish function: :ShowAndRunOrig, type: "symbol (list <string>, map <string, map <string, any>>, term, string, string, string, map)"
     publish function: :InitNull, type: "void (string)"
     publish function: :StoreNull, type: "void (string, map)"
+
+  private
+
+    def widgets_in_contents(contents)
```

Comment:
but it is not correct for more complex contents as you do not do searching in deep. As I write below problem is old ruby bindings...just install newer or at least use newer term code - https://github.com/yast/yast-ruby-bindings/blob/master/src/ruby/yast/term.rb#L14